app.py

```diff
@@ -0,0 +1,30 @@
+import flask
+import requests
+import os
+
+app = flask.Flask(__name__)
+
+MY_API_SECRET_KEY = "x95xd2dbxc3x175md5"
+
+BASE_URL = "https://api.nytimes.com/svc/search/v2/articlesearch.json"
+
+params = {
+    "q": "braves",  # Query keywords
+    "api-key": os.getenv(MY_API_SECRET_KEY),
+}
```

Comment:
This is a multiline comment. 
Put api call inside the input function so when we get a new call on the page load